tests/end_to_end.rs

```diff
@@ -1,37 +1,47 @@
-use assert_cmd::cmd::Command;
-use std::time::Duration;
+use anyhow::{Context, Result};
+use std::path::{Path, PathBuf};
 use tempfile::TempDir;
 
-#[test]
-fn test_file_inputs() {
-    let root = TempDir::new().unwrap();
+fn output_of_default_job(root: &TempDir, rbt_dot_roc: &Path) -> Result<PathBuf> {
+    let current_dir = rbt_dot_roc
+        .parent()
+        .context("failed to get parent of the target rbt.roc")?;
+    let filename = rbt_dot_roc
+        .file_name()
+        .context("failed to get the filename of the target rbt.roc")?;
 
-    Command::new("roc")
+    let output = std::process::Command::new("roc")
         .arg("run")
         .arg("--linker=legacy")
-        .arg("rbt.roc")
+        .arg(filename)
         .arg("--")
         .arg("--root-dir")
         .arg(root.path().display().to_string())
-        .current_dir("tests/end_to_end/file_inputs")
-        .timeout(Duration::from_secs(10))
-        .assert()
-        .success();
-
-    let store_path = root
-        .path()
-        .join("store")
-        .read_dir()
-        .expect("could not read dir")
-        .flatten()
-        .filter(|item| {
-            item.file_type()
-                .map(|ftype| ftype.is_dir())
-                .unwrap_or(false)
-        })
-        .map(|item| item.path())
-        .next()
-        .expect("expected at least one directory in the store path");
+        .arg("--print-root-output-paths")
+        .current_dir(current_dir.display().to_string())
```

Comment:
No timeout?

Reply:
this moved from using `assert_cmd` to the regular `std::process::Command`, which doesn't have a timeout method. Think it's worth switching back to get that?

Reply:
Ok. Didn't realize the change here. This is fine as well.